requirements.txt

```diff
@@ -9,9 +9,9 @@ joblib>=1.1.0
 gt4sd-molformer>=0.1.1
 keras>=2.3.1,<2.11.0
 keybert>=0.7.0
+markdown-it-py<3.0.0,>=2.2.0
```

Comment:
If this is a requirement of the core package it should be added to the `setup.cfg` too, in order to keep things in sync as much as possible

Reply:
it's not a direct requirement of core hence the decision to leave it out, it's a dependency that gets installed with a messy bound in conda builds.